### DISCUSSION TITLE
Question regarding non-credited work being used in another repository/removed credit.

### DISCUSSION THREAD

**Discussion by Scuhf-git** [community/community#56291] [Other Feature Feedback, Questions, & Ideas]
Question

Hi! I was just wondering if there was any violation ToS wise if someone was to use the work from another repository and then actively remove credit from said work?

Just curious since a situation like this is currently happening and i'd like to better inform myself before taking any actions against it since i'm not full sure. The work that was used HAD proper credit in comments on the code and PR's but was removed by the people who've merged it with their codebase, we've asked for the credit to be put back but they've refused, should I be worried or do anything or is this fine?

**Comment by shayneoneill** [comment]
Whats the license of the software in question say? The response from yourself can be anything from "Hey, dont be a dick, throw me a credit please" to "expect a call from my lawyer". But it all depends on the license or lack thereof. A *lack* of license also has more complicated implications than you might presume. If you consider it egrarious enough your best bet is to ask a real lawyer instead of the internet, particularly if it poses either a financial threat to you or an existential threat to your company or project.

**Reply by Scuhf-git** [reply]
The repository license says it's a GNU Affero General Public License v3.0, but someones also told me that my work might somehow also be under CC BY-SA 3? Take that with a grain of salt though, but I am sure it falls under the GNU Affero license as the repository my work in states.

I've read through CC BY-SA 3 a little and it seems like if it's true that my work falls under that license i'd have to be given credit.

**Reply by Scuhf-git** [reply]
I really don't want anything legal to take place over this, just proper credit.

**Reply by shayneoneill** [reply]
If the license in your repo says its AGPL3, then its most certainly AGLP3. And that would exclude. CC BY-SA3 because they aren't compatible.

**Reply by Scuhf-git** [reply]
That's what I was thinking but someone told me about the CC BY-SA 3 license and confused me a bit, thanks for clearing that up though because well, as you can tell i'm not the most savvy in this type of stuff lol.

Purely just artist/spriter trying to hopefully get credit back on something that they've removed it from.

**Comment by Scuhf-git** [comment]
I've talked this over and hopefully we'll be receiving credit now.

**Comment by shayneoneill** [comment]
Thats an excellent outcome (And probably what a lawyer would have sought)

Good work.
